main.go

```diff
@@ -57,14 +59,27 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&development, "development", true, "Configures the logger to use a development config (stacktraces on warnings, no sampling)")
+
 	opts := zap.Options{
 		Development: development,
 	}
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	// set log level from environment variable if provided and no flag was set
+	if logLevel := os.Getenv("LOG_LEVEL"); logLevel != "" && opts.Level == nil {
```

Comment:
This line sends a flag https://github.com/aiven/aiven-operator/pull/982/files#diff-493d41a6cf8a4d52b86d40c55fb49d39b69c3f29a9042406b17f8a0a2056128aR67
We shouldn't expect an env variable here.

Reply:
I mean, why then we need `zap-log-level`? It is not handled. Am I missing something? :(

Reply:
discussed